examples/website/point-cloud/app.js

```diff
@@ -6,14 +6,10 @@ import {COORDINATE_SYSTEM, OrbitView, LinearInterpolator} from '@deck.gl/core';
 import {PointCloudLayer} from '@deck.gl/layers';
 
 import {LASWorkerLoader} from '@loaders.gl/las';
-// import {PLYWorkerLoader} from '@loaders.gl/ply';
 
 // Data source: kaarta.com
 const LAZ_SAMPLE =
   'https://raw.githubusercontent.com/visgl/deck.gl-data/master/examples/point-cloud-laz/indoor.0.1.laz';
-// Data source: The Stanford 3D Scanning Repository
-// const PLY_SAMPLE =
-//   'https://raw.githubusercontent.com/visgl/deck.gl-data/master/examples/point-cloud-ply/lucy800k.ply';
```

Comment:
Removing because this doesn't really work - the PLY file referenced here contains a mesh instead of point cloud, and the PointCloudLayer is unable to render it correctly.